src/index.html

```diff
@@ -4,19 +4,111 @@
     <meta charset="UTF-8" />
     <meta
       name="viewport"
-      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
+      content="width=device-width, initial-scale=1.0"
     />
-    <meta
-      http-equiv="X-UA-Compatible"
-      content="ie=edge"
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+      crossorigin="123"
     />
-    <title>Moyo header</title>
     <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@300;400;500;700;900&display=swap"
       rel="stylesheet"
-      href="./style.css"
     />
+    <link
+      rel="stylesheet"
+      href="style.css"
+    />
+    <title>Moyo header</title>
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <div class="wrapper">
+      <main class="main">
+        <header class="header">
+          <a
+            href="123"
+            class="logo"
+          >
+            <img
+              src="images/logo.png"
+              alt="MoYo Logo"
+            />
+          </a>
+          <nav class="nav">
+            <ul class="nav__list">
+              <li class="nav_item">
+                <a
+                  class="nav_link is-active"
+                  href="123"
```

Comment:
The `href` attribute of the `a` tag should not contain spaces and should start with the `#` symbol if it is intended to be an anchor link.